remix-gql-tailwind/app/components/RepoNavigation/RepoNavigation.tsx

```diff
@@ -36,7 +37,7 @@ function RepoNavigation() {
                     >
                       {crumb}
                     </Link>
-                    <span className={styles.separator}>/</span>
+                    <span key={i} className={styles.separator}>/</span>
```

Comment:
You need to fix this one too. Generally though, the better key would be to just use the crumbPath or href values.

Reply:
thanks..I used the href